nuxeo-mongodb-core/src/main/java/org/nuxeo/mongodb/core/MongoDBSerializationHelper.java

```diff
@@ -42,6 +42,12 @@
 
     public static final String MONGODB_ID = "_id";
 
+    public static final String MONGODB_SET = "$set";
+
+    public static final String MONGODB_INC = "$inc";
```

Comment:
We shouldn't declare mongodb operations. There're classes to perform this operation in an elegant way. See Updates/Filters...